examples/quickstart/00-rbac.json

```diff
@@ -0,0 +1,70 @@
+{
+   "apiVersion": "v1",
+   "items": [
+      {
+         "apiVersion": "v1",
+         "kind": "Namespace",
+         "metadata": {
+            "name": "heptio-sonobuoy"
+         }
+      },
+      {
+         "apiVersion": "v1",
+         "kind": "ServiceAccount",
+         "metadata": {
+            "labels": {
+               "component": "sonobuoy"
+            },
+            "name": "sonobuoy-serviceaccount",
+            "namespace": "heptio-sonobuoy"
+         }
+      },
+      {
+         "apiVersion": "rbac.authorization.k8s.io/v1beta1",
```

Comment:
It's not easy right now, but it's certainly on the roadmap for the CLI.

One way to do it right now is to use [ext vars][ev], which are passed in via command line. Your build process would involve interrogating the API server, getting capabilities, and passing a flag `--ext-str` with a value to indicate as such.

[ev]: http://jsonnet.org/docs/stdlib.html#ext_vars